pkg/asset/cluster/tfvars.go

```diff
@@ -79,6 +81,7 @@ func (t *TerraformVariables) Dependencies() []asset.Asset {
 		new(rhcos.BootstrapImage),
 		&bootstrap.Bootstrap{},
 		&machine.Master{},
+		&machine.Worker{},
```

Comment:
So the workers are created by machine-api and the ignition is available to the k8s Machine object from a secret already created.

Can you provide information on why the ignition for worker is being passed to the terraform using tfvars asset.

Reply:
I wanted to create them eagerly and not to wait for the machine-api to kick in.

Reply:
@rgolangh for parity with other platforms and making sure the installer can slowly move towards creating only bootstrap-host, it is important that the terraform only create bootstrap-host and control-plane hosts today.

So I would like that you drop creating workers and depend on machine-api to perform that task.

Reply:
ack